shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/encrypt/config/query-with-cipher.yaml

```diff
@@ -22,7 +22,7 @@ dataSources:
 rules:
 - !ENCRYPT
   tables:
-    t_account:
+    T_account:
```

Comment:
Can we keep the encrypt config and add some more test case for encrypt capitalization?

Reply:
Hi, I just mixed this logic table name with upper and lower cases, to simulate that issue, for testcases.

Reply:
OK, i got your mean.